src/scales/scale.time.js

```diff
@@ -465,6 +469,8 @@ module.exports = Scale.extend({
 		var me = this;
 		var options = me.options;
 		var time = options.time || (options.time = {});
+		var adapterOptions = options.adapters;
+		var adapter = me._adapter = new adapters._date((adapterOptions && adapterOptions.date) || {});
```

Comment:
See previous comment:

```javascript
var adapter = me._adapter = new adapters._date((options.adapters || {}).date);
```

Reply:
We could also add `adapters: {}` in the time scale defaults and use `options.adapters.date` directly.

Reply:
I took the latter suggestion. I think it's better to pass in just the date adapter options rather than all adapter options

Reply:
> I think it's better to pass in just the date adapter options rather than all adapter options

I think the same and it wasn't the case in the first suggestion.